### DISCUSSION TITLE
Apenas me dieron mi usuario para entrar al Teams

### DISCUSSION THREAD

**Discussion by OsirisSosa** [LaunchX-InnovaccionVirtual/CursoIntroPython#265] [Q&A] [answered]
Hola a todos 

Se que suena raro, pero apenas el viernes en la tarde me dieron los accesos al Teams y al programa LaunchX. Estoy leyendo los videos, pero estoy viendo que ya varios terminaron y que el Lunes hoy ya recibieron un correo para el FrontEnd, etc.

Mi duda en específico es.
¿Aun puedo terminar las katas y seguir?

La sherpa @FernandaOchoa dice que hasta este Viernes y aquí en otro lado. ¿Me pueden orientar?

**Comment by GloriaCoralCerecedo** [comment]
Si, la Sherpa lo menciono de hecho esta en el último live, tienes hasta el Lunes 21 me parece que hasta las 20:00 hrs de ese día. Te sugiero que veas los live.

**Comment by carlos-gmx3** [comment, marked as answer]
La mission commander tiene la última palabra. Todavia estas a tiempo de entrar a la misión frontend. 
El formulario para entregar las katas  apenas fue liberado el día de hoy a eso de las 13hrs y estará abierto hasta el lunes 21. Esto lo puedes confirmar en el live del día viernes 11 de Febrero

**Reply by OsirisSosa** [reply]
Muchas gracias !
